spec/controllers/api/v1/base_controller_spec.rb

```diff
@@ -0,0 +1,9 @@
+describe Api::V1::BaseController do
+  it 'sets responder' do
+    expect(described_class.responder).to eq(InfinumJsonApiSetup::JsonApi::Responder)
```

Comment:
Whats the idea here, to show an example + test the implementation or just test the implementation? A one-off controller defined inside the spec file might satisfy the latter one without requiring a separate file. Not sure if its worth it to add the magic tho

Reply:
As per https://github.com/infinum/infinum-json-api-setup/pull/21#issuecomment-878290610 I'm leaning towards integration testing.

THB, for now, this is really a weird spec, that checks if you followed README correctly. In the future, I'd like to move that part of the README configuration into the install generator.